<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnlockedWindow</class>
 <widget class="QMainWindow" name="UnlockedWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menedżer haseł</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hasła</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>781</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>520</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>521</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuń</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>521</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyświetl</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionChangeMasterPassword"/>
    <addaction name="actionLock"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuPlik"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>Nowy</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Otwórz</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Zapisz jako</string>
   </property>
  </action>
  <action name="actionChangeMasterPassword">
   <property name="text">
    <string>Zmień hasło główne</string>
   </property>
  </action>
  <action name="actionLock">
   <property name="text">
    <string>Zablokuj</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
